recipes/pixman/all/conandata.yml

```
@@ -1,6 +1,8 @@
 sources:
   "0.40.0":
-    url: "https://www.cairographics.org/releases/pixman-0.40.0.tar.gz"
+    url:
+    - "https://gitlab.freedesktop.org/pixman/pixman/-/archive/pixman-0.40.0/pixman-pixman-0.40.0.tar.gz"
+    - "https://www.cairographics.org/releases/pixman-0.40.0.tar.gz"
```

```suggestion
    url: "https://www.cairographics.org/releases/pixman-0.40.0.tar.gz"
```
the gitlab archive does not have the same sha256, it is automatically generated by gitlab, contrary to the release tarball on cairographics.org

---

I thought I didn't commit that change, I saw it locally too, but it seems like commit -a was faster than me. Thank you for catching it, I'll remove it.

---

Done.